### DISCUSSION TITLE
how to create a banner?

### DISCUSSION THREAD

**Discussion by Metro-City** [emukidid/swiss-gc#574] [Q&A]
hi, I wanted to know how I can create the banner at F-Zero AX that appears in the menu with "no banner" thk

**Comment by emukidid** [comment]
I'm not sure if there's anything super recent but back in the day there was a tool to convert TGA to BNR files, that'd get you a banner with some generic text and then you'd have to edit it in something (or hex edit it) to fix up the rest. You might have better luck asking on the gc-forever forums, or searching around for tools on there.
